src/test/kotlin/com/exactpro/th2/common/schema/TestDictionaryLoad.kt

```diff
@@ -39,4 +42,40 @@ class TestDictionaryLoad {
         }
     }
 
+    @Test
+    fun `test folder load dictionaries by alias`() {
+        val factory = CommonFactory.createFromArguments("-c", "src/test/resources/test_load_dictionaries")
+
+        Assertions.assertDoesNotThrow {
+            factory.loadDictionary("test_alias_2").use {
+                assert(String(it.readAllBytes()) == "test file")
+            }
+        }
+    }
+
+    @Test
+    fun `test folder load all dictionary aliases`() {
+        val factory = CommonFactory.createFromArguments("-c", "src/test/resources/test_load_dictionaries")
+        val expectedNames = listOf("test_alias_1", "test_alias_2", "test_alias_3", "test_alias_4")
+        val names = factory.loadDictionaryAliases()
+        Assertions.assertEquals(4, names.size)
+        Assertions.assertTrue(names.containsAll(expectedNames))
+    }
+
+    @Test
+    fun `test folder load single dictionary`() {
+        val factory = CommonFactory.createFromArguments("-c", "src/test/resources/test_load_dictionaries")
+
+        Assertions.assertThrows(IllegalStateException::class.java) {
+            factory.loadDictionary()
+        }
+
+        val customSettings = FactorySettings().apply { dictionaryAliasesDir = Path.of("src/test/resources/test_load_dictionaries/single_dictionary") }
+        val customFactory = CommonFactory(customSettings)
+
+        customFactory.loadDictionary().use {
+            assert(String(it.readAllBytes()) == "test file")
+        }
```

Comment:
I don't think that we should mix checking of valid and invalid cases in a single test method